### ISSUE TITLE
Add graphics to github pages

### ISSUE THREAD

**Issue by pohmelie** [Neon22/micropython-examine-mem-maps#2]
It looks like you have static csv/html files. So this can be hosted by github.

**Comment by Neon22** [comment]
Yes they are static, I had to do a lot of research to find a way to reference images in the readme.md file. I'll look into how to make references using github.

Currently I'm focussing on making it work when you download to a local machine.
So paths are relative to current zip directory.
I might leave it that way as I don't think I can have it work both ways...
